src/ts/AppBar/index.tsx

```diff
@@ -1,21 +1,20 @@
-import * as React from 'react'
-import { Link } from 'react-router'
-import { AppBar } from 'material-ui'
-import Menu from '../Menu'
-import Search from '../Search/Location'
-import muiThemeVG from '../muiThemeVG'
-import debug from 'debug'
-var log = debug('app:AppBar')
+import * as React from 'react';
+import { Link } from 'react-router';
+import { AppBar } from 'material-ui';
+import Menu from '../Menu';
+import { default as Search } from '../Search/Location';
```

Comment:
because of linter - it's allow us to import module only with the same name as in exported file (here is `Location`)